rest/service/favoritePageService.go

```diff
@@ -49,6 +53,15 @@ func UpdateFavoritePage(favoritePage models.FavoritePage) error {
 	return database.DB.Model(&models.FavoritePage{}).Where("pathname = ?", favoritePage.Pathname).Update("favorite", favoritePage.Favorite).Error
 }
 
+func debugFavoritesEntry(accountId uint, payload models.FavoritePage) {
+	c := config.Get()
+	for _, i := range c.DebugConfig.DebugFavoriteIds {
+		if i == fmt.Sprint(accountId) {
+			logrus.Warningln(fmt.Sprintf("\n_____\nDEBUG_FAVORITES_ACCOUNT_ID: %d\nDEBUG_FAVORITES_PATH: %s\nDEBUG_FAVORITES_FLAG: %s\n_____", accountId, payload.Pathname, fmt.Sprint(payload.Favorite)))
```

Comment:
Should this be `logrus.Debugln(...)` ?

Reply:
@BlakeHolifield we would have to set the log level to debug which would undo the effort updating the logs. :(